CHANGELOG.md

```diff
@@ -74,6 +74,9 @@ and `WindowEvent::HoveredFile`.
 - On Mac, implement `DeviceEvent::Button`.
 - Change `Event::Suspended(true / false)` to `Event::Suspended` and `Event::Resumed`.
 - On X11, fix sanity check which checks that a monitor's reported width and height (in millimeters) are non-zero when calculating the DPI factor.
+- On Windows, fix window rectangle not getting set correctly on high-DPI systems.
+
+# 0.20.0 Alpha 1
```

Comment:
This section was erroneously added in earlier versions of this branch, but it's since been moved to the unreleased section where it should be. We should remove this here.